src/piconim.nim

```diff
@@ -18,7 +18,9 @@ proc builder(program: string, output = "") =
       removeFile(file)
 
   # compile the nim program to .c file
-  let compileError = execCmd(fmt"nim c -c --nimcache:{nimcache} --gc:arc --cpu:arm --os:any -d:release -d:useMalloc ./src/{program}")
+  let nimcmd = fmt"nim c -c --nimcache:{nimcache} --cpu:arm --os:any -d:useMalloc ./src/{program}"
+  echo nimcmd
```

Comment:
Should we be always echoing this or was this for debugging?

Reply:
Personally, I think we should be echoing by default, with perhaps a `--quiet` option, as we're already getting a whole bunch of make/gcc output anyways, and it helps devs potentially understand what's going on with their build.

I'm not against adding a "Compiler command:" prefix however.